crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -747,9 +748,18 @@ impl RnAppWindow {
 
         // Clipboard paste
         action_clipboard_paste.connect_activate(clone!(@weak self as appwindow => move |_, _| {
-            let canvas = appwindow.active_tab_wrapper().canvas();
+            let canvas_wrapper = appwindow.active_tab_wrapper();
+            let canvas = canvas_wrapper.canvas();
             let content_formats = appwindow.clipboard().formats();
 
+            let last_contextmenu_pos = canvas_wrapper.last_contextmenu_pos().map(|vec2| {
```

Comment:
This introduces a bug: When performing a subsequent paste from the app-menu or via `<ctrl>+v` it still uses this position. To fix this, we probably need another action that is only for the context-menu paste.